.github/workflows/ci.yml

```
@@ -123,7 +123,7 @@ jobs:
     strategy:
       fail-fast: ${{ !contains(github.event.pull_request.labels.*.name, 'CI-no-fail-fast') }}
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
```

It looks like numpy 1 never shipped compiled wheels for 3.13 (not really a surprise), I think we can skip supporting numpy 1 on 3.13:

```suggestion
        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12"]
```

https://pypi.org/project/numpy/1.26.4/#files